gtk4/src/auto/cell_area.rs

```diff
@@ -230,7 +230,7 @@ pub trait CellAreaExt: 'static {
         &self,
         context: &impl IsA<CellAreaContext>,
         widget: &impl IsA<Widget>,
-        snapshot: &Snapshot,
+        snapshot: &impl IsA<Snapshot>,
```

Comment:
And this here can only take a `GtkSnapshot`. With any other `GdkSnapshot` it would be wrong

Reply:
Well, there can't be any other kind of GdkSnapshot other than GtkSnapshot. This is basically caused by moving GtkSnapshot methods to a trait, can't do much about it unless we manually implement everything which is alot :)

Reply:
> Well, there can't be any other kind of GdkSnapshot other than GtkSnapshot.

Until the GTK people decide to add one and all this falls apart, yes :)